polaris.shopify.com/content/design/layout.md

```diff
@@ -34,7 +28,6 @@ Each layout component encapsulates CSS styles commonly found in the Shopify admi
 
 You can browse all [layout and structure components](/components/layout-and-structure), as well as view usage guidelines and examples for each component.
 
----
```

Comment:
I accidentally deleted my comment that said we could remove this. But sorry this was an error on my part, we should add this divider back 😅